client/js/_author/components/common/oea_editor.jsx

```diff
@@ -142,6 +142,8 @@ export class OeaEditor extends React.Component {
   }
 
   getEditorContent(media) {
+    if (!_.isUndefined(media.error)) { return ''; }
```

Comment:
This makes it so it doesn't insert a placeholder with the `src='undefined'`. We could come up with a way to pop up an error to say that something broke, but I figured this would be better than just showing a placeholder that looks like it works.